src/app/music/page.js

```
@@ -0,0 +1,104 @@
+                     index={index}
+                     album_object={album}
+                     onAnimationEnd={({ album, state }) =>
+                        state === 'active'
+                           ? setAlbumInDrawer(album)
+                           : setAlbumInDrawer(null)
+                     }
+                     active={activeIndex === index}
+                     setActive={(e) =>
+                        e ? setActiveIndex(index) : setActiveIndex(null)
+                     }
+                  />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)